packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -67,13 +69,14 @@ function installFrontierDependencies(appPath, appName, ownPath) {
       'storybook:build': 'NODE_ENV=development build-storybook -c .storybook -o build',
       lint: 'eslint src/',
       'lint:fix': 'eslint src/ --fix',
-      test: `eslint src/ && ${packageJson.scripts.test}`,
     }
     packageJson.scripts = { ...packageJson.scripts, ...additionalScripts }
     delete packageJson.scripts.eject
-    packageJson.eslintConfig = {
-      extends: ['@fs/eslint-config-frontier-react'],
+    packageJson.eslintConfig = { extends: ['@fs/eslint-config-frontier-react'] }
+    packageJson.husky = {
+      hooks: { 'pre-commit': 'lint-staged', 'pre-push': 'npm run lint && CI=true npm test' },
     }
+    packageJson['lint-staged'] = { '*.js': ['suppress-exit-code eslint --fix', 'git add'] }
```

Comment:
how do we feel about config in package.json vs config in dot files or external config files?
Does lint-staged have an dotfile/external config file option?

I don't like really long messy package.json files, but it's a style thing, and having tons of dot files in the root of the project can be problematic too.

This particular one isn't onerous, but I've seem some package.jsons with dozens of config lines for various dependencies, so just looking for future patterns.

Reply:
Personally, it doesn't bother me in the slightest. If we start getting in the hundreds of lines, then let's worry about splitting stuff into dotfiles I guess.